flink-table/flink-table-planner/pom.xml

```diff
@@ -329,6 +347,7 @@ under the License.
 							<include>commons-codec:commons-codec</include>
 							<include>commons-io:commons-io</include>
 							<include>org.apache.commons:commons-math3</include>
+							<include>org.apache.commons:commons-text</include>
```

Comment:
Is this really needed?

Reply:
tests shown that we can live without it
removed